### ISSUE TITLE
The Trained model is not providing any response

### ISSUE THREAD

**Issue by bhadreshpsavani** [iusztinpaul/hands-on-llms#66]
I trained model end to end and once i deployed it on gradio for chat assistant, It was giving empty reponse for all the query

**Comment by iusztinpaul** [comment]
Hello,

Have you trained it for the same number of epochs as in our config or more?

**Comment by bhadreshpsavani** [comment]
Hello @iusztinpaul,

I trained it with same number of epochs.

**Comment by iusztinpaul** [comment]
We had the same behavior when training for too many epochs. This is probably because there is not enough variety in the dataset, and the model overfits. 

Here are 2 ways to debug this issue:
1. Try to use our model to run the inference to see that everything works fine
2. Lower the number of epochs
3. If lowering the number of epochs doesn't work, try also to lower the learning rate.

Let me know if this helps.
